doc/api/dgram.md

```diff
@@ -95,6 +95,21 @@ Tells the kernel to join a multicast group at the given `multicastAddress` and
 one interface and will add membership to it. To add membership to every
 available interface, call `addMembership` multiple times, once per interface.
 
+### socket.addSourceSpecificMembership(sourceAddress, groupAddress[, multicastInterface])
+<!-- YAML
+added: REPLACEME
+-->
+* `sourceAddress` {string}
+* `groupAddress` {string}
+* `multicastInterface` {string}
+
+Tells the kernel to join a source-specific multicast channel at the given 
+`sourceAddress` and `groupAddress`, using the `multicastInterface` with the
+`IP_ADD_SOURCE_MEMBERSHIP` socket option. If the `multicastInterface` argument
+is not specified, the operating system will choose one interface and will add
+membership to it. To add membership to every available interface, call 
+`addSourceSpecificMembership` multiple times, once per interface.
```

Comment:
minor style nit: s/addSourceSpecificMembership/addSourceSpecificMembership()`  (add the `()` to the end when mentioning functions)

Reply:
Fixed, I emulated the style of the document to also reference the object e.g. `socket.addSourceSpecificMembership()`.